vessel-runtime/src/main/java/com/textnow/android/vessel/VesselImpl.kt

```diff
@@ -51,44 +51,64 @@ import kotlin.reflect.KClass
  * @param cache Optional [VesselCache]. Built-ins: [DefaultCache] and [LruCache] [Default: null]
  */
 class VesselImpl(
-        private val appContext: Context,
-        private val name: String = "vessel-db",
-        private val inMemory: Boolean = false,
-        private val allowMainThread: Boolean = false,
-        private val callback: VesselCallback? = null,
-        private val cache: VesselCache? = null
-): Vessel {
+    private val appContext: Context,
+    private val name: String = "vessel-db",
+    private val inMemory: Boolean = false,
+    private val allowMainThread: Boolean = false,
+    private val callback: VesselCallback? = null,
+    private val cache: VesselCache? = null,
+    private val profile: Boolean = false,
```

Comment:
Missed adding a `@param` comment for the new `profile` param :]

Reply:
Good catch, will add.